python/ccf/clients.py

```diff
@@ -605,13 +592,12 @@ def _call(
         body: Optional[Union[str, dict, bytes]] = None,
         http_verb: str = "POST",
         headers: Optional[dict] = None,
-        signed: bool = False,
         timeout: int = DEFAULT_REQUEST_TIMEOUT_SEC,
         log_capture: Optional[list] = None,
     ) -> Response:
         description = ""
         if self.description:
-            description = f"{self.description}{signed * 's'}"
+            description = f"{self.description}{bool(self.sign) * 's'}"
```

Comment:
```suggestion
            description = f"{self.description}{int(self.sign) * 's'}"
```

`self.sign` is already boolified on assignment, so we don't need to recast it here (`self.sign * 's'`) should work too. But I think casting to int is more readable.

Reply:
Yes, good point, this is a leftover from before sign was there. I think this can be simplified further actually.